base/src/main/java/com/smartdevicelink/proxy/rpc/UnsubscribeVehicleData.java

```diff
@@ -867,4 +868,21 @@ public void setOEMCustomVehicleData(String vehicleDataName, Boolean vehicleDataS
 	public Boolean getOEMCustomVehicleData(String vehicleDataName){
 		return getBoolean(vehicleDataName);
 	}
+
+	/**
+	 * Sets a status value for WindowStatus.
+	 * @param status a Boolean value
+	 */

```

Comment:
```suggestion
/**
     * Sets the windowStatus.
     *
     * @param windowStatus See WindowStatus
     * @since SmartDeviceLink 7.0.0
     */
```
Please use the [RPC generator](https://github.com/smartdevicelink/sdl_java_suite/tree/master/generator) in the sdl_java_suite project to ensure the RPC matched the spec exactly. The generator can also be used to update the XML comments at the top of this file